admin_guide/scheduling/scheduler.adoc

```diff
@@ -212,27 +213,40 @@ they have to be explicitly specified in the scheduler configuration file.
 === Default Scheduler Policy
 The default scheduler policy includes the following predicates:
 
-. NoVolumeZoneConflict
-. MaxEBSVolumeCount
-. MaxGCEPDVolumeCount
-. MatchInterPodAffinity
-. NoDiskConflict
-. GeneralPredicates
-. PodToleratesNodeTaints
-. CheckNodeMemoryPressure
-. CheckNodeDiskPressure
+. `NoVolumeZoneConflict`. 
+. `MaxEBSVolumeCount`. Checks the maximum volume limit for a node.
+. `MaxGCEPDVolumeCount`. Checks the maximum PD volume limit for a node.
+. `MatchInterPodAffinity`. Checks if the pod affinity/antiaffinity rules permit the pod.
+. `NoDiskConflict`. Checks the volume requested by a pod is available.
+. `GeneralPredicates`
```

Comment:
I think this could be further split into non-critical and essential. In fact, essential predicates are the ones through which resource requests go through to see if they fit.

Reply:
@ravisantoshgudimetla Which ones are essential? Hard to tell from the code. 

This PR lists essential predicates as: PodFitsResources, PodFitsHostPorts, HostName, MatchNodeSelector, which we don't list here. 
https://github.com/kubernetes/kubernetes/pull/50990
Are we listing only non-critical here?